### DISCUSSION TITLE
SSR - ESM Compability

### DISCUSSION THREAD

**Discussion by ObjectAssign** [Maronato/vue-toastification#387] [Q&A]
Hi, im running an application which are looking for a more modern Toast notification...

The initial pick is/was vue-toastnotification, it looks awesome and is easy to setup..

However, im using Laravel, Inertia, Vue3 Composition API with Vite and SSR....

And the {useToast} import fails my SSR badly...

I get this error:

import pkg from 'vue-toastification';
const { useToast } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)
Node.js v18.17.1

Ive tried the solution in the log, however then useToast is just not recognised as a function, i've tried many different solutions by now and nothing comes up satisfying...

Am i doing something wrong, or is it just not compatible with my setup?

**Comment by Lionelsan95** [comment]
Hello, 

what if you just to do in one line ? 

`import { useToast } from 'vue-toastification';`

**Reply by ObjectAssign** [reply]
Thats how i did at first, the compiler told me i could try the other approach, it is not ESM compatible :)

I made it work by hacking it into my app.ts inertia setup, now it seems to work without SSR complaning :)
